frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -117,6 +151,12 @@ pub mod pallet {
 	pub type AssociatedVaults<T: Config> =
 		StorageValue<_, BoundedBTreeSet<T::VaultId, T::MaxAssociatedVaults>, ValueQuery>;
 
+	/// An asset whitelisted by Instrumental
+	/// The corresponding Pool to invest the whitelisted asset into
```

Comment:
```suggestion
	/// An asset whitelisted by Instrumental.
	///
	/// The corresponding Pool to invest the whitelisted asset into.
```
Rustdoc uses [Markdown](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#markdown), so if you want to write two separates lines/paragraphs, then add empty line between them.

Reply:
@belousm, you changed status to resolved, but what do you want to do here? Because it will be rendered as:

> An asset whitelisted by Instrumental The corresponding Pool to invest the whitelisted asset into

It will be without empty line and without punctuation marks.

Reply:
Bump! @belousm